installer/templates/static/brunch/brunch-config.js

```diff
@@ -51,6 +51,13 @@ exports.config = {
       ignore: [/web\/static\/vendor/]
     }
   },
+
+  modules: {
+    autoRequire: {
+      'web/static/js/app.js': ['app']
```

Comment:
The left side was correct, you had to change the right side to mirror what you removed from the layout.